src/array/cuda/coo_sort.cu

```diff
@@ -0,0 +1,155 @@
+/*!
+ *  Copyright (c) 2020 by Contributors
+ * \file array/cuda/coo_sort.cc
+ * \brief Sort COO index
+ */
+#include <dgl/array.h>
+#include "../../runtime/cuda/cuda_common.h"
+#include "./utils.h"
+
+namespace dgl {
+
+using runtime::NDArray;
+
+namespace aten {
+namespace impl {
+
+///////////////////////////// COOSort_ /////////////////////////////
+
+template <DLDeviceType XPU, typename IdType>
+void COOSort_(COOMatrix* coo, bool sort_column) {
+  CHECK(sizeof(IdType) == 4) << "CUDA COOSort does not support int64.";
```

Comment:
Do we plan to support int64 COOSort in the future?  If no, why?

Reply:
Currently we use cusparse which only supports int32_t. We should support that in the future using the RadixSort routine from CUB. I'll leave a TODO there.